test/integration/image-component/default/test/index.test.js

```diff
@@ -590,6 +616,7 @@ describe('Image Component Tests', () => {
           target: 'serverless',
           experimental: {
             enableBlurryPlaceholder: true,
+            staticImages: true
```

Comment:
Why `enableBlurryPlaceholder` but not `enableStaticImages` (or vice versa)?

Reply:
Standardized